R/shortcuts.R

```diff
@@ -30,6 +30,6 @@ cheat <- function(sheet) {
 
   sheet <- paste0(sheet, ".pdf")
 
-  fpath <- system.file("extdata/cheatsheets", sheet, package = "integral") %>% paste0("'", ., "'")
```

Comment:
This weird paste was something that was needed for some platforms that can't handle spaces in a path

Reply:
![image](https://user-images.githubusercontent.com/25590353/202296659-cd60d9ec-94be-4836-a1d3-b0fa41af07a4.png)

Reply:
Oh funky - so I fixed it for windows but broke it for mac (and presumably linux). Whoops!